src/vs/base/browser/keyboardEvent.ts

```diff
@@ -25,13 +25,17 @@ function extractKeyCode(e: KeyboardEvent): KeyCode {
 	} else if (browser.isFirefox) {
 		switch (keyCode) {
 			case 59: return KeyCode.Semicolon;
-
+			case 60:
+				if (platform.isLinux) { return KeyCode.IntlBackslash; }
```

Comment:
I couldn't try out this `IntlBackslash` thing, but the rest is correct

Reply:
:+1: It is present only on ISO keyboards:

![image](https://user-images.githubusercontent.com/5047891/231888178-e670e97c-8d77-4f9d-b458-b61fca9a3718.png)